crates/bevy_text/src/lib.rs

```diff
@@ -76,7 +76,18 @@ use bevy_sprite::SpriteSystem;
 /// When the `bevy_text` feature is enabled with the `bevy` crate, this
 /// plugin is included by default in the `DefaultPlugins`.
 #[derive(Default)]
-pub struct TextPlugin;
+pub struct TextPlugin {
+    /// Attempts to load system fonts if [true].
+    ///
+    /// Supports Windows, Linux and macOS.
+    ///
+    /// System fonts loading is a surprisingly complicated task,
+    /// mostly unsolvable without interacting with system libraries.
+    /// It will simply scan some predefined directories.
```

Comment:
This needs to links to further documentation explaining exactly which directory :) It's fine if that's in an external crate though.